ambry-replication/src/main/java/com/github/ambry/replication/BackupCheckerThread.java

```diff
@@ -142,43 +139,42 @@ protected void applyUndelete(MessageInfo remoteBlob, RemoteReplicaInfo remoteRep
    * @param remoteReplicaInfo Info about remote replica from which we are replicating
    */
   @Override
-  protected void applyPut(MessageSievingInputStream validMessageDetectionInputStream, RemoteReplicaInfo remoteReplicaInfo) {
+  protected void applyPut(MessageSievingInputStream validMessageDetectionInputStream,
+      RemoteReplicaInfo remoteReplicaInfo) {
     throw new IllegalStateException("We should not be in applyPut since we override fixMissingStoreKeys");
```

Comment:
Could we remove all the fixMissingStoreKeys in comments since we override {@link #afterHandleReplicaMetadataResponse} instead?